brume/stack.py

```diff
@@ -40,17 +40,32 @@ def _make_parameters(params_list):
     return [{'ParameterKey': k, 'ParameterValue': v} for k, v in params_list.items()]
 
 
-def _outputs_for(stack):
+def _stack_walker(client, outputs, stack, collector):
```

Comment:
Can you provide a docstring for this method?

Reply:
done